src/docs-assembler/navigation.yml

```diff
@@ -37,60 +37,112 @@ toc:
   #################
   # RELEASE NOTES #
   #################
-  # I didn't touch this section (yet?)
   - toc: release-notes
```

Comment:
This is the structure @KOTungseth defined for the release notes section. This will need to be validated.